docs/modules/tiered-storage/pages/overview.adoc

```diff
@@ -25,3 +25,13 @@ Tiered Storage uses the following:
 == Not Yet Supported
 
 - Data recovery from persisted data: When a Hazelcast member has Tiered Storage enabled and it fails or shuts down, you are not able to use the local device files of Tiered Storage for data recovery. Data recovery will be supported in the future releases.
+
+- TTL expiration is not supported by Tiered-Storage-backed maps: methods like `IMap.put` will throw `UnsupportedOperationException` if a TTL value is provided. If the default TTL setting is present in the map configuration, `InvalidConfigurationException` will be thrown during the node startup.
+
+- Max-Idle expiration is not supported by Tiered-Storage-backed maps: methods like `IMap.put` will throw `UnsupportedOperationException` if a Max-Idle value is provided. If the default Max-Idle setting is present in the map configuration, `InvalidConfigurationException` will be thrown during the node startup.
+
+- Eviction is not supported: if an eviction policy is configured for a Tiered-Storage-backed map, `InvalidConfigurationException` will be thrown during the node startup.
+
+- Data Persistence and Tiered Storage are mutually exclusive features: if both are enabled simultaneously, `InvalidConfigurationException` will be thrown during the node startup.
+
+- SQL is not supported for Tiered-Storage-backed maps: `UnsupportedOperationException` is thrown if SQL query is being executed. Use Predicate API instead.
```

Comment:
```suggestion
- SQL is not supported for Tiered-Storage-backed maps. `UnsupportedOperationException` is thrown when SQL query is executed. To avoid this, use Predicate API instead.
```